finality-aleph/src/aggregation/mod.rs

```diff
@@ -38,9 +214,13 @@ where
         self.0.next().await
     }
 
-    fn send(&self, data: D, recipient: Recipient) -> Result<(), NetworkError> {
-        self.0.send(data, recipient).map_err(|e| match e {
-            SendError::SendFailed => NetworkError::SendFail,
+    fn send(
+        &self,
+        data: D,
+        recipient: legacy_aleph_bft::Recipient,
+    ) -> Result<(), CurrentNetworkError> {
```

Comment:
Another current <-> legacy difference, not sure if intended

Reply:
once again intended for now at least (aggregator crate uses fixed recipient from aleph-bft-types)